api/v1/types.go

```diff
@@ -216,10 +216,6 @@ type Driver struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Snapshot Classes"
 	SnapshotClass []SnapshotClass `json:"snapshotClass,omitempty" yaml:"snapshotClass"`
 
-	// ForceUpdate is the boolean flag used to force an update of the driver instance
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Force update"
-	ForceUpdate bool `json:"forceUpdate,omitempty" yaml:"forceUpdate"`
```

Comment:
Did you regenerate the CRD manifests with the respective Makefile targets? Also please run e2e and attach the results.

Reply:
Yes regenerate the CRD , please recheck i add the e2e result